obspy/signal/cross_correlation.py

```diff
@@ -76,7 +76,44 @@ def _xcorr_slice(a, b, shift, domain='freq'):
     return c[mid - shift:mid + shift + len(c) % 2]
 
 
-def correlate(a, b, shift, demean=True, normalize=True, domain='freq'):
+def _normxcorr(a, b, shift, domain='freq'):
```

Comment:
Please document the arguments.

Reply:
Can do, I was following the `_xcorr_slice` and `_xcorr_pad` precedent, shall I document them while I'm there (copy and paste)?

Reply:
Parts of our code base are not yet up to the standard and even for private functions it definitely helps if they are properly documented especially with not immediately obvious parameters. This improves maintainability for the next person touching the code.